docker/Dockerfile

```diff
@@ -36,17 +40,20 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive \
     curl \
     smartmontools \
     tzdata \
-    && update-ca-certificates \
-    &&  case ${TARGETARCH} in \
-            "amd64")  S6_ARCH=amd64  ;; \
-            "arm64")  S6_ARCH=aarch64  ;; \
-        esac \
+    && update-ca-certificates
+RUN case ${TARGETARCH} in \
+        "amd64")  S6_ARCH=amd64  ;; \
+        "arm64")  S6_ARCH=aarch64  ;; \
+    esac \
     && curl https://github.com/just-containers/s6-overlay/releases/download/v${S6VER}/s6-overlay-${S6_ARCH}.tar.gz -L -s --output /tmp/s6-overlay-${S6_ARCH}.tar.gz \
     && tar xzf /tmp/s6-overlay-${S6_ARCH}.tar.gz -C / \
-    && rm -rf /tmp/s6-overlay-${S6_ARCH}.tar.gz \
-    && curl -L https://dl.influxdata.com/influxdb/releases/influxdb2-${INFLUXVER}-${TARGETARCH}.deb --output /tmp/influxdb2-${INFLUXVER}-${TARGETARCH}.deb \
+    && rm -rf /tmp/s6-overlay-${S6_ARCH}.tar.gz
+RUN curl -L https://dl.influxdata.com/influxdb/releases/influxdb2-${INFLUXVER}-${TARGETARCH}.deb --output /tmp/influxdb2-${INFLUXVER}-${TARGETARCH}.deb \
+    && ln -s /usr/bin/false /bin/false \
+    && ln -s /usr/bin/bash /bin/bash \
```

Comment:
what's the intention here?

Reply:
This is related to the merged /usr required in bookworm. /bin/false doesn't exist so when the user is created the useradd command throws a warning. The simple solution was to create a symbolic link, create the user, the delete the symbolic link. No more warning is thrown.